nengo/builder/signal.py

```diff
@@ -210,7 +234,12 @@ def size(self):
     @property
     def strides(self):
         """(tuple) Strides of data in bytes."""
-        return self.initial_value.strides
+        return None if self.sparse else self.initial_value.strides
+
+    @property
+    def sparse(self):
+        """(bool) Whether the signal is sparse."""
+        return not npext.is_array_like(self.initial_value)
```

Comment:
For sparse, `initial_value` has to be `spmatrix` or `SparseMatrix`, right? I'd prefer to check for those, a) in case `is_array_like` changes to include e.g. `spmatrix`, and b) so it's clearer reading the code what constitutes a sparse matrix.

Reply:
Makes sense, changed!